Dockerfile

```diff
@@ -1,17 +1,25 @@
 FROM alpine:latest as build
 
-ADD . /sslh
-
 RUN \
   apk add \
     gcc \
     libconfig-dev \
     make \
     musl-dev \
     pcre2-dev \
-    perl && \
-  cd /sslh && \
-  make sslh-select && \
+    perl \
+    perl-dev \
+    perl-app-cpanminus \
+    git
+
+WORKDIR /sslh
+RUN git clone https://github.com/yrutschle/conf2struct && \
+	cpanm --force Conf::Libconfig && \
+	make -C conf2struct checker && \
+	make -C conf2struct install
```

Comment:
how hard is this requirement for conf2struct? Is this something that should be packaged individually first?

In either case, the container is becoming quite developer heavy, and would benefit from having a BUILDER stage  where everything is compiled (e.g. what we have now) and then a final container stage, where everything gets copied into.